src/compas_robots/model/link.py

```diff
@@ -148,14 +148,15 @@ def __from_data__(cls, data):
         return cls(origin, mass, inertia)
 
 
-class LinkItem(object):
+class LinkItem(Data):
     def __init__(self):
+        super(LinkItem, self).__init__()
```

Comment:
since `Data` already defines an attribute `name` I'd add it here as optional kwarg and pass it up to `super`.